pontoon/base/models.py

```diff
@@ -1812,14 +1820,6 @@ def is_asymmetric(self):
     def __str__(self):
         return '%s: %s' % (self.project.name, self.path)
 
-    def save(self, *args, **kwargs):
-        super(Resource, self).save(*args, **kwargs)
-
-        if self.deadline and self.project.deadline:
-            if self.deadline < self.project.deadline:
-                self.project.deadline = self.deadline
-                self.project.save()
```

Comment:
Why do you remove this? I understand that it doesn't work as expected, but does it mean the intention is was wrong or useless? Don't we need to replace this with something that works?

Reply:
We never used this function. The idea was to set the project deadline based on the resource deadline, but that's wrong, because the project deadline in this case should be different for each locale (based on what files they have enabled and what's their progress). I can file a bug, but we could also first see if there's a need for such feature, now that we'll actually start using resource deadlines.

Reply:
OK, sounds good to me!